tests/utils_test.py

```diff
@@ -177,7 +177,7 @@ def load_and_clean_json(location):
     data = json.loads(content)
 
     if 'creationInfo' in data:
-        del(data['creationInfo'])
+        del (data['creationInfo'])
```

Comment:
Is `del` special somehow to earn a space between the next bracket?

Reply:
Yeah, looks like it's a special reserved keyword. I didn't even notice this, autoformat did it...